dependencies.gradle

```diff
@@ -1,24 +1,24 @@
 def versions = [
-        buildtools               : '3.5.3',
-        icappsGradlePlugin       : '3.0.1',
-        icappsArch               : '0.5.0',
+        buildtools               : '4.1.1',
+        icappsGradlePlugin       : '3.0.2',
+        icappsArch               : '0.5.2',
         icappsAnnotations        : '0.1.1',
-        icappsAnnotationProcessor: '0.1.2',
-        kotlin                   : '1.3.60',
-        detekt                   : '1.4.0',
-        lifecycle                : '2.2.0',
-        appcompatV7              : '1.2.0-alpha01',
-        recyclerView             : '1.2.0-alpha01',
+        icappsAnnotationProcessor: '0.1.3',
+        kotlin                   : '1.4.20',
+        detekt                   : '1.14.1',
+        lifecycle                : '2.3.0-beta01',
+        appcompatV7              : '1.3.0-alpha02',
+        recyclerView             : '1.2.0-alpha06',
         cardView                 : '1.0.0',
-        constraintLayout         : '2.0.0-beta4',
-        material                 : '1.2.0-alpha03',
-        retrofit                 : '2.7.1',
-        okhttp                   : '3.12.1', // OkHttp 3.12.1 is the last version that works on API <21. Bump version if desired
-        niddler                  : '1.0.1',
-        dagger                   : '2.25.4',
-        moshi                    : '1.9.2',
+        constraintLayout         : '2.1.0-alpha1',
+        material                 : '1.3.0-alpha04',
+        retrofit                 : '2.9.0',
+        okhttp                   : '4.10.0-RC1', // OkHttp 3.12.1 is the last version that works on API <21. Bump version if desired
```

Comment:
Remove the comment, if you bump this version anyways

Reply:
It is still helpful until < 21 support will die completely